src/engine/helper/pka_helper.c

```diff
@@ -181,6 +181,103 @@ static void free_operand(pka_operand_t *operand)
     free(operand);
 }
 
+/* ECC related helpers */
```

Comment:
coding style issue: I think in this file comments are as follow:

  // One line comment

  //
  // Multi-line comment
  //

Reply:
Ack, Thanks.